Documentation/contributing/development/dev_setup.rst

```diff
@@ -517,10 +517,10 @@ Minor version
    date with these changes.
 
 #  Update documentation files:
-   - Documentation/concepts/kubernetes/compatibility.rst
-   - Documentation/concepts/kubernetes/requirements.rst
    - Documentation/contributing/testing/e2e.rst
-   - Documentation/gettingstarted/istio.rst
+   - Documentation/network/istio.rst
+   - Documentation/network/kubernetes/compatibility.rst
+   - Documentation/network/kubernetes/requirements.rst
```

Comment:
nit: We would typically commit separate changes like this to another commit since it's unrelated to the overall k8s X.Y bump.

Reply:
that makes sense, i'll keep that in mind for next time :+1: